docs/API.md

```diff
@@ -216,6 +216,7 @@ query HelloWorld {
 |------------------|------------------|----------------------------------------|
 | service          | http-client      | The service name for this integration. |
 | splitByDomain    | false            | Use the remote endpoint host as the service name instead of the default. |
+| validateStatus   | `code => true`   | Callback function to determine if an HTTP response should be recorded as an error. It should take a status code as its only parameter and return `true` for success or `false` for errors. Requests where no response is received (e.g. connection failures, timeouts) will always be considered to be errors.
```

Comment:
Happy to remove this, but I think this is a useful point of clarification to make it clear that this doesn't configure how "no response" is handled.

Reply:
I think it's redundant since validate**Status** implies a response from the remote endpoint.

Reply:
Fair enough! I'll ✂️ this. 

Perhaps a better place for this would be in some more general documentation for the HTTP plugin? I've found that I need to delve into the code to answer questions about the specifics.

Reply:
Definitely feel free to add any information you think is relevant to the top of the plugin documentation :)

Reply:
I'll do this in a separate PR.